.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--require ui_auto_helper
```

Comment:
This is going to pull it into unit tests too

Reply:
Good point, it's removed.
However, given this is a shared project now then this will mean an undesirable _require_ in each spec file for UI tests or alternatively to read command-line arguments for the spec helper location.

Reply:
`rails_helper` now gets added with scaffolded specs, and honestly I do agree with you, I prefer to leverage the `.rspec` file to tidy the head of each spec, see https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#testing-utilities

I'd sooner follow that convention, drop `rails_helper` and let your page model framework specs do the explicit include. Alternatively we can call `rspec` to run your UI tests with `-r`; however we'd also have to exclude your specs from a normal run. 

Swings and roundabouts.